client/rest-high-level/src/main/java/org/elasticsearch/client/SecurityRequestConverters.java

```diff
@@ -67,6 +68,15 @@ static Request changePassword(ChangePasswordRequest changePasswordRequest) throw
         return request;
     }
 
+    static Request getUsers(GetUsersRequest getUsersRequest) {
+        RequestConverters.EndpointBuilder builder = new RequestConverters.EndpointBuilder()
+            .addPathPartAsIs("_xpack/security/user");
```

Comment:
`/_xpack/security/user` - > `/_security/user` since https://github.com/elastic/elasticsearch/pull/36293 and https://github.com/elastic/elasticsearch/pull/36379 are merged

Reply:
:+1: